### DISCUSSION TITLE
Dynamic Parallelism in CUTLASS?

### DISCUSSION THREAD

**Discussion by dukallis** [NVIDIA/cutlass#1959] [Q&A] [answered]
Dynamic Parallelism (DP) seems to me a very exotic feature of CUDA and I've seen it used rarely in some papers.

Do I understand it right that PDL and persistent kernels are possible due to Dynamic Parallelism of CUDA? Does it really the case that one kernel call some other one or do I misunderstand something really bad?

**Comment by thakkarV** [comment, marked as answer]
No, we don't call other kernels from a kernel anywhere
